src/migrate_dependencies.js

```diff
@@ -0,0 +1,23 @@
+async function migrate({
+  artifacts,
+  deployer,
+  network,
+}) {
+  if (network === "development" || network == "coverage") {
```

Comment:
When checking the network name, please take into account that truffle migrate when running in dry-run uses a different name (appends `-fork` to the name)